### PR TITLE
Testing todolist

### PR DESCRIPTION
**Pull request by Jessekay**
**THROUGH THIS MILESTONE WE IMPLEMENTED THE FOLLOWING:**

- Used jest framework for testing.
- Created a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test.
- Mocked a storage object to "imitate" localStorage operations (only if you are using jest < 24.x, in newer versions localStorage is mocked automatically).
- Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.
- We grouped our tests using the describe() method.

### CODE REVIEW

File: modules/functions.test.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const localStorage = new LocalStorage('./localstorage');
+
+describe('removeTask', () => {
+  let cleanup;
+
+  beforeEach(() => {
+    const dom = new JSDOM('<!DOCTYPE html><div id="container"></div>', {
+      url: 'http://localhost',
+    });
+    global.document = dom.window.document;
```
Comment (V-Blaze) [inline modules/functions.test.js:16]: - [ ]  Great job writing the test for add and remove functionality, :+1: :100: I think there are some improvements you need to make here before moving to the next milestone, :+1: 
- [ ] Currently, your tests are failing, it's giving an error in the console, I'll suggest you run the test one more time and try to fix the error following the suggestion from the console.
- Screenshot For reference:

Screenshot from 2023-05-10 20-17-52

File: README.md [hunk 2]
```diff
@@ -1 +1,142 @@
...
\ No newline at end of file
+<a name="readme-top"></a>
+
+all the comments and instructions!
+
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```
Comment (V-Blaze) [inline README.md:9]: - [OPTIONAL] I noticed that this image link is broken, this is a placeholder logo added by Microverse, I'll suggest you remove this as it's not needed in your readme and the broken logo makes your readme look a bit unprofessional. :smile: :airplane:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @binodbhusal, @yashodhicy, @Jessekay  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights

Professional Pull Request :heavy_check_mark: 
 Project 1. has a test file ([..].test.js) for a file containing the add item and delete item functions that would be tested. :heavy_check_mark: 
Tests are grouped using the description() method.✔


 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/functions.test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    tasks.length = 0;
+    localStorage.clear();
+  });
+
+  test('adds a new task to the list', () => {
+    // Set up a mock task entry and call the addList function
+    // const taskEntry = { value: 'Task 1' };
```
Comment (1GORDON) [inline modules/functions.test.js:64]: Great job team 🤝

- [ ] [Optional] Kindly remove all commented code from your project as this is not regarded as a good practice.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Jessekay, @binodbhusal, & @yashodhicy ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed GitHub Flow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**